index.html

```diff
@@ -3433,7 +3438,7 @@ <h3><dfn>Writing content</dfn></h3>
           </li>
           <li>
             Let |record:NDEFRecord| be the result of <a>parse an NDEF record</a>
-            on |ndef|.
+            given |ndef| and `"root"`.
```

Comment:
Why pass `root` here? Does it mean nothing?

Reply:
Now passing `""`.